cypress/e2e/home_page.cy.ts

```diff
@@ -1,13 +1,33 @@
 describe("The Home Page", () => {
```

Comment:
Overall, much happier with this test suite. All stubs are actually happening, and these feel like true e2e tests.